linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -24,8 +24,9 @@ use tokio::process::Command;
 pub struct LocalKubernetesNetConfig {
     pub network: Network,
     pub testing_prng_seed: Option<u64>,
+    pub num_initial_validators: usize,
+    pub num_shards: usize,
     pub binaries_dir: Option<PathBuf>,
-    pub cluster_id: Option<u32>,
```

Comment:
Since we're gonna be creating multiple kind clusters now, passing the `cluster_id` doesn't make sense anymore since we'll have to create multiple of these